src/app/components/Layout/Footer/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+// styles
+import { WrapperFooter, CopyrightText } from "./coponents";
+
+const Footer: React.FunctionComponent = () => {
+  return (
+    <React.Fragment>
```

Comment:
no need to use Fragment here. Also you can write shorter form of arrow function here (without "return").